calyx-py/calyx/gen_exp.py

```diff
@@ -8,9 +8,8 @@
 )
 from calyx.utils import float_to_fixed_point
 from math import factorial, log2
-from fud.stages.verilator import numeric_types
+from calyx.numeric_types import FixedPoint
```

Comment:
It is a great outcome here that the `calyx-py` library doesn't need to depend on fud for this. That direction of dependency was always extremely weird.